applications: nrf_desktop: Add release_fast_pair for nRF54LS05 DK (#29905)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

nRF Desktop
-----------

* Added:

>   * Support for the ``nrf54lc10dk/nrf54lc10a/cpuapp`` and  ``nrf54ls05dk/nrf54ls05a/cpuapp`` board targets.

Reviewer: ```suggestion
  * Support for the ``nrf54lc10dk/nrf54lc10a/cpuapp`` and ``nrf54ls05dk/nrf54ls05a/cpuapp`` board targets.
```

* Added support for the ``nrf54lc10dk/nrf54lc10a/cpuapp`` board target.
* Added support for the ``nrf54ls05dk/nrf54ls05a/cpuapp`` board target.
* Added the ``release_fast_pair`` build type for the ``nrf54ls05dk/nrf54ls05a/cpuapp`` and ``nrf54ls05dk/nrf54ls05b/cpuapp`` board targets.

>   The configuration enables Google Fast Pair mouse support with MCUboot direct-xip and software-based image signature verification.

Reviewer: ```suggestion
* Added:
 
  * Support for the ``nrf54lc10dk/nrf54lc10a/cpuapp`` and  ``nrf54ls05dk/nrf54ls05a/cpuapp`` board targets.
  * The ``release_fast_pair`` build type for the ``nrf54ls05dk/nrf54ls05a/cpuapp`` and ``nrf54ls05dk/nrf54ls05b/cpuapp`` board targets.
    The configuration enables Google Fast Pair mouse support with MCUboot direct-xip and software-based image signature verification.
```

>   The configuration enables Google Fast Pair mouse support with MCUboot direct-xip and software-based image signature verification.

Reviewer 2: ```suggestion
  The configuration acts as a HID mouse with Fast Pair support.
  It uses MCUboot in direct-xip mode with software-based image signature verification.
```

Author: applied
